docs/en/observability/synthetics-projects-workflow.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[synthetics-project-monitors]]
+// lint ignore params
+= Working with project monitors
+
+++++
+<titleabbrev>Working with project monitors</titleabbrev>
+++++
+
+beta[]
+
+Projects are the most powerful and sophisticated way to configure synthetic monitors in the Elastic Stack. Projects let you define your infrastructure as code, more commonly known as IaaC or git-ops. With project monitors you organize your YAML configuration and Javascript/Typescript defined monitors on the filesystem, use Git for version control, and deploy via a CLI tool, usually executed on a CI/CD platform.
+
+
+== Getting started
```

Comment:
Did you intend for each section to be on its own page (see [the preview](https://observability-docs_2353.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-project-monitors.html))? If not, you can add a `[discrete]` tag above the h2 headings (`==`).

```suggestion
[discrete]
== Getting started
```

Reply:
Not at all! This is much better